src/en/rizzcomic/build.gradle

```diff
@@ -2,8 +2,8 @@ ext {
     extName = 'Rizz Comic'
     extClass = '.RizzComic'
     themePkg = 'mangathemesia'
-    baseUrl = 'https://rizzcomic.com'
-    overrideVersionCode = 2
+    baseUrl = 'https://rizzfables.com/'
```

Comment:
```suggestion
    baseUrl = 'https://rizzfables.com'
```

Reply:
I just noticed and pushed a commit addressing that 😅